common/scala/src/main/resources/logback.xml

```diff
@@ -11,8 +11,10 @@
 
   <!-- Kafka -->
   <logger name="org.apache.kafka" level="ERROR" />
+
+  <variable name="WHISK_LOG_LEVEL" value="${WHISK_LOG_LEVEL:-INFO}" />
   
-  <root level="info">
+  <root level="${WHISK_LOG_LEVEL}">
```

Comment:
I guess what @rabbah was after, is that this could now be changed to:

```
// entirely omit the <variable> line
<root level="whisk.loglevel">
```

While defining:

```
-e CONFIG_whisk_loglevel=...
```

on the components.

Your call on what you prefer.

In any case: Is the double-definition even necessary in your case? Couldn't it be just:

```
<root level="${WHISK_LOG_LEVEL:-INFO}">
```
?

Reply:
@markusthoemmes Changed (see my other comment)